atest/testdata/standard_libraries/builtin/verify.robot

```diff
@@ -293,6 +297,44 @@ Should Contain With Non-String Values
     Should Contain    ${dict}    a
     Should Contain    ${dict}    c
 
+Should Contain Any
+    [Documentation]    FAIL Message: 'abcdefg' does not contain any of ''A', 'X' or 'Ф''
```

Comment:
There are extra quotes around the list of items but that's due to the `_get_string_msg` helper unconditionally adding them. I guess the helper could get a new config parameters like `quote_str1=True` and `quote_str2=True` to control that, and the new keywords could call it like `quote_str2=False`. What do you think?

Somewhat related to this, I'd prefer the `_get_string_msg` to add a full stop at the end of the default message. That's totally unrelated to this PR (the change require changing lot of tests) and, unless you want to help with that too, I can take care of that myself.